js/modules/timeline.src.js

```diff
@@ -181,6 +180,31 @@ undocumentedSeriesType('timeline', 'line',
 
             Series.prototype.init.apply(series, arguments);
 
+            addEvent(series, 'afterTranslate', function () {
+                var lastPlotX,
+                    closestPointRangePx = Number.MAX_VALUE;
+
+                series.points.forEach(function (point) {
+                    // Set the isInside parameter basing on the real point
+                    // visibility, in order to avoid showing hidden points
+                    // in drawPoints method.
+                    point.isInside = point.visible;
+
+                    // New way of calculating closestPointRangePx value, which
+                    // respects the real point visibility is needed.
+                    if (point.visible) {
+                        if (defined(lastPlotX)) {
```

Comment:
Can we use?

```
if (point.visible && defined(lastPlotX)) {
   ...
}
```

Reply:
It's not good way out, because `lastPlotX` always would be `undefined`, and `closestPointRange` will never be set.

Reply:
It's not good way out, because `lastPlotX` would always be `undefined`, and `closestPointRange` will never be set.

Reply:
Ok, makes sense for me!